### PR TITLE
create meeting notes for Oct 14

### LINKED ISSUE #48: Post Client Meeting Minutes for Oct. 14
Describe what needs to get done
Markdown file of the minutes from the client meeting

### PR DESCRIPTION
**Pull request by vnajack**
closes #48

### CODE REVIEW

**Review by edouarde1 — APPROVED** [review]
Thanks for taking the notes during the meeting, I really like that you included the to-do list.

File: docs/meetings/client/2022_10_14.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- Team A:
+  - Edouard Eltherington
+  - Veronica Jack
+  - Logan Parker
+
+***Attendence Comments***
+
+- Matt Kuelker is writing a midterm, so he was unable to attend
+- Students from other teams present as well
+  - Team B only had 1 group member present
```
Comment (LoganParker) [inline docs/meetings/client/2022_10_14.md:23]: Team C was missing 1 member

File: docs/meetings/client/2022_10_14.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+- They proposed alternate milestones for their group:
+  1. Basic phoneme detection + usage of Kukarella TTS
+  2. Generate a speaking portrait with audio
+  3. API endpoint integration with Kukarella allowing use of preset avatars
+  4. User-friendly method of uploading custom avatars
+
+## To-do List
+
+1. Send Nazim a complied list (all groups together) of what we all need access to and the deadline
```
Comment (LoganParker) [inline docs/meetings/client/2022_10_14.md:77]: We still need to send over the email for our premium access

File: docs/meetings/client/2022_10_14.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+2. Go into the Google Sheets file that Nazim shared and request edit access (Veronica received edit access immediately after the meeting)
+
+3. Once you have edit access, adjust the milestone calendar to reflect each team's milestones.
```
Comment (LoganParker) [inline docs/meetings/client/2022_10_14.md:81]: We have edit access!

File: docs/meetings/client/2022_10_14.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+ images. Nazim and Erik said that this is acceptable.
+
+- We went over our DFDs (level 0 and level 1) to ensure that the requirements we understood are correct, and both Nazim
+and Erik confirmed that we are on the right track.
+
+- We asked about access to making requests from the Kukarella system to receive the audio file, language, and user text.
+There currently is no Kukarella API for us to call, so we will need some other way to access the audio, language, and text.
+
+- We asked if there are currently any 2D assets that we can have access to. The image assets that they currently have are
+animation files from their other app, so they're not still images and they do NOT have mouth images at this time
```
Comment (LoganParker) [inline docs/meetings/client/2022_10_14.md:43]: We should add this to our request :)

**Review by LoganParker — APPROVED** [review]
This looks great! thank you for taking these notes :)

File: docs/meetings/client/2022_10_14.md (hunk 1 above)
Comment (vnajack) [reply docs/meetings/client/2022_10_14.md:23]: will add as a note in a re-commit, thanks for catching that!

File: docs/meetings/client/2022_10_14.md (hunk 3 above)
Comment (vnajack) [reply docs/meetings/client/2022_10_14.md:81]: We do! I kept the third to-do because we need to do that in our team meeting!

**Pull request merged**